nox/_options.py

```diff
@@ -142,11 +142,18 @@ def _envdir_merge_func(
     return command_args.envdir or noxfile_args.envdir or ".nox"
 
 
-def _sessions_default() -> list[str] | None:
-    """Looks at the NOXSESSION env var to set the default value for sessions."""
-    nox_env = os.environ.get("NOXSESSION")
-    env_sessions = nox_env.split(",") if nox_env else None
-    return env_sessions
+def default_env_var_list_factory(env_var: str) -> Callable[[], list[str] | None]:
+    """Looks at the env var to set the default value for a list of env vars.
+
+    Args:
+        env_var (str): The name of the environment variable to look up.
+    """
```

Comment:
Would ideally like a `Returns` section here, particularly as we're returning a callable which is fine and a good solution for this, just probably not what most people will expect

Reply:
@FollowTheProcess thanks for the feedback! Done: d1887d5413d6c9d37caa98a455a2243bcc452ec9